cmd/argo/commands/list.go

```diff
@@ -64,6 +64,34 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows",
+		Example: `# List all workflows:
+  argo list
+
+# Show workflows from all namespaces:
```

Comment:
```suggestion
# List all workflows from all namespaces:
```

I missed this in my previous review, but we should consistently use the verb "list"; this line was the odd one out